src/Congo.Blazor/Models/Product.cs

```diff
@@ -0,0 +1,21 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace Congo.Blazor.Models
+{
+    public class Product
```

Comment:
This is identical to the ProductResponse found in the .Contracts project

Reply:
I tried changing it to `ProductResponse` in Contracts project and I get errors on `init` only properties. How would I get pass this?

Reply:
If I am correct I believe @pobiega wanted to use different request models for the consumer of the API than the commands used by the handlers.
So could add the InsertProductRequest model to the Requests folder in the .Contracts project